locale/es/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,1083 @@
+#: tracpro/templates/polls/pollrun_participation.html:173
+msgid "Reporter Group"
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:31
+msgid "Send Message..."
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:36
+msgid "Restart..."
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:51
+msgid "Send Message"
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:67
+#, python-format
+msgid ""
+"\n"
+"                          All participants in %(region)s (%(count)s)\n"
+"                        "
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:71
+#, python-format
+msgid ""
+"\n"
+"                          All participants (%(count)s)\n"
+"                        "
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:80
+#, python-format
+msgid ""
+"\n"
+"                          Respondents in %(region)s (%(count)s)\n"
+"                        "
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:84
+#, python-format
+msgid ""
+"\n"
+"                          Respondents (%(count)s)\n"
+"                        "
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:93
+#, python-format
+msgid ""
+"\n"
+"                          Non-respondents in %(region)s (%(count)s)\n"
+"                        "
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:97
+#, python-format
+msgid ""
+"\n"
+"                          Non-respondents (%(count)s)\n"
+"                        "
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:119
+msgid "Send"
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:134
+msgid "Restart Non-respondents"
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:144
+#, python-format
+msgid ""
+"\n"
+"                Restart poll for %(count)s non-respondents in %(region)s?\n"
+"              "
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:148
+#, python-format
+msgid ""
+"\n"
+"                Restart poll for %(count)s non-respondents?\n"
+"              "
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:159
+msgid "Restart"
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:182
+msgid "Partial response"
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:185
+msgid "Full response"
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:188
+msgid "Completion"
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:216
+msgid "No group"
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:218
+msgid "No region"
+msgstr ""
+
+#: tracpro/templates/polls/pollrun_participation.html:238
+msgid "Total"
+msgstr ""
+
+#: tracpro/templates/polls/response_by_pollrun.html:10
+#, python-format
+msgid ""
+"\n"
+"        These are the responses for <strong>%(region)s</strong>\n"
```

Comment:
Neither am I :smile_cat: As it turns out, yes you can:

https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#pluralization